tests/unit/specs/components/common/TmBalance.spec.js

```diff
@@ -40,7 +40,21 @@ describe(`TmBalance`, () => {
       overview: {
         totalStake: 3210,
         liquidStake: 1230,
-        totalRewards: 1000.45
+        totalRewards: 1000.45,
+        rewards: [
+          {
+            amount: 1,
+            denom: `utoken1`
```

Comment:
here is the issue. you implemented the API wrong. the rewards need to be delivered in a view token

Reply:
It is in the works luniehq/lunie#351

I will change now tests